benchmarks/functions/__init__.py

```diff
@@ -18,6 +18,30 @@ class FunctionBenchmark(BenchmarkBase):
     # Repeat the test for 10 times instead of 3 (`timeit.default_repeat`).
     repeat = 10
 
+    def _convert_to_variable(self, x):
+        if x is None:
+            return None
+        elif isinstance(x, (int, float, bool, str)):
+            return x
+        elif isinstance(x, (list, tuple)):
+            return ([self._convert_to_variable(i) for i in x])
+        else:
+            return chainer.Variable(x)
+
+
+    # Make the format of outputs tuple of xp.ndarray/chainer.Variable.
+    def _normalize_outputs(self, x):
+        def rec(x):
+            if isinstance(x, (list, tuple)):
+                ret = []
```

Comment:
Use variable name other than `i` as it sounds like an index. (e.g., `elem`)

Reply:
Fixed.